enterprise/server-enterprise/src/main/java/org/neo4j/server/enterprise/OpenEnterpriseNeoServer.java

```diff
@@ -118,6 +122,28 @@ public OpenEnterpriseNeoServer( Config config, Database.Factory dbFactory, Graph
         }
     }
 
+    @Override
+    public void start() throws ServerStartupException
+    {
+        super.start();
+        registerDiscoverableBoltRoutingURI();
+    }
+
+    private void registerDiscoverableBoltRoutingURI()
+    {
+        if ( getConfig().get( EnterpriseEditionSettings.mode ) == Mode.CORE )
+        {
+            URI boltRoutingUri = DiscoverableURIs.discoverableBoltUri( "bolt_routing", getConfig(),
```

Comment:
Will "bolt_routing" be returned in JSON response?
I think it is weird to have JSON response keys defined like this as strings in such high-level components as `OpenEnterpriseNeoServer`

Reply:
Ack, agree; I'll do this as a server module instead